lib/book.rb

```diff
@@ -0,0 +1,19 @@
+class Book
+  attr_reader :title
+
+  def initialize(info)
+    @author_first_name = info[:author_first_name]
+    @author_last_name = info[:author_last_name]
+    @title = info[:title]
+    @publication_date = info[:publication_date]
+  end
+
+  def author
+    "#{@author_first_name} #{@author_last_name}"
+  end
+
+  def year
+    @publication_date.chars.last(4).join
```

Comment:
This works, but I'd rather not have you create a new array data structure to get at those last 4 characters. #chars creates an array

Reply:
Also, we wanted this method to be called publication_year